plaso/parsers/ips_parser.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+"""Parser for ips formatted log files. Each file is a single event."""
```

Comment:
`ips` => `iOS IPS` also explain what IPS is

Reply:
IPS files are used by Apple in iOS and MacOS. So I'll leave is as IPS?

Reply:
explain in the docstring what is IPS short of? the name overlaps with Cisco IOS Intrusion Prevention System 
So I maybe `Apple JSON Crash format (IPS)`